docs/hard-fork-upgrade/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Hard Fork Upgrade
+sidebar_label: Hard Fork Upgrade
```

Comment:
Berkley Upgrade again

Reply:
In fact replace all "Hard Fork" with Berkeley Upgrade ... I won't go through whole doc with these comments ;)